target/linux/ramips/dts/ArcherC20v4.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
```

Comment:
You can include TL-WR84XN.dtsi and remove:
chosen, memory, spi, ehci, ohci
In "wmac" leave only "mtd-mac-address-increment",
In "Ethernet" remove "mtd-mac-address"

ps. Maybe its good time to change filename from TL-WR84XN.dtsi to TPLINK_8M.dtsi ?

Reply:
Please don't include the TL-WR84XN.dtsi or at least not as long as the file is named TL-WR84XN.dtsi. Such DTS files are a pain to read.

TPLINK_8M.dtsi is way better. But it shouldn't happen in context of this PR. I'm, fine to merge an extra PR which does these kind of optimizations.

Reply:
> Please don't include the TL-WR84XN.dtsi
Ok